### PR TITLE
Review Cleanup

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

Reviewing the upcoming Check glossary and other PRs has resulted in some random cleanup/consistency things in this PR

- use the CDS components `LastRequestDate` and `RequestsCount` for media cards to keep things more consistent and DRY
- allow themes props for `RequestsCount` component
- start to align around rejected media instead of unmatched/detached media
- cleanup some layout issues on the media cards and remove material ui components in favor of css
- sort prop types and remove es lint disable on touched files

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

manual verification

## Things to pay attention to during code review

no real visual or functionality changes here. just some more consistent styles by using shared components

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)
- [ ] If I touched a file that included an eslint-disable-file header, I updated the code such that the disabler can be removed

### CODE REVIEW

**Comment by brianfleming** [comment]
reviewers, not sure what is up with the smoke test failures. I ran them twice and they failed for different reasons...flaky?

File: src/app/components/feed/FeedImportDialog.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Relay from 'react-relay/classic';
 import PropTypes from 'prop-types';
-import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
```
Comment (amoedoamorim) [inline src/app/components/feed/FeedImportDialog.js:4]: can we also remove `injectIntl` ?

File: src/app/components/feed/FeedItemMediaList.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
-/* eslint-disable react/sort-prop-types */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { QueryRenderer, graphql } from 'react-relay/compat';
 import Relay from 'react-relay/classic';
-import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { injectIntl } from 'react-intl';
```
Comment (amoedoamorim) [inline src/app/components/feed/FeedItemMediaList.js:5]: is there a reason to keep `injectIntl` if we're not using the `intl` prop?

File: src/app/components/feed/FeedImportDialog.js (hunk 1 above)
Comment (brianfleming) [reply src/app/components/feed/FeedImportDialog.js]: seems like it, the build didn't pester me about removing it so I missed it! Thanks

File: src/app/components/feed/FeedItemMediaList.js (hunk 2 above)
Comment (brianfleming) [reply src/app/components/feed/FeedItemMediaList.js]: same as previous comment, resolved now

**Review by amoedoamorim — APPROVED** [review]
LGTM. Love to see a bunch of duplicated strings go!

**Pull request merged**